modules/install-central-operator.adoc

```diff
@@ -10,7 +10,18 @@ The main component of {product-title} is called Central. You can install Central
 
 [IMPORTANT]
 ====
-When you install {product-title} for the first time, you must first install the `Central` custom resource because the `SecuredCluster` custom resource installation is dependent on certificates that Central generates.
```

Comment:
OpenShift comes with an out of the box proxy that we integrate with and leverage. So this setting does not apply entirely and all pages to this page can be dropped.

Reply:
If you are referring to the cluster-wide proxy this does not allow setting the proxy for individual components as it sets it the proxy config on all pods in a cluster.
WDYTH about documenting both ways and recommending the cluster-wide proxy?
And is it possible to hide the customize config in our CRs behind a button if the cluster-wide proxy is recommended? 

Here are the docs to the cluster-wide proxy:
https://docs.openshift.com/container-platform/4.9/networking/enable-cluster-wide-proxy.html

Reply:
@gaurav-nelson - For the proxy settings this is reasonably simple. Everything that you added here should be removed. Then we'd call out that RHACS is setup to respect the cluster proxy and point them to the cluster wide proxy documentation that @SimonBaeumer is mentioning.